src/module_TOUCHSCREEN/TouchscreenApp.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Salt Lake Community College
+ * Spring 2017 Semester
+ * CSIS-2810 Computer Architecture
+ * Instructor: Jon McGowan
+ * 
+ */
+
+package module_TOUCHSCREEN;
```

Comment:
I've not ever dealt with an Arduino or the compile/upload process that it goes through. Its possible having both Java and C++ code inside the same source directory might cause issues, but I really have no idea. If it works then this will be a good structure. If it doesn't then we can figure out a structure that will and isolate each language from the other. We can worry about that when we get to it.